docs/features/workspaces.md

```diff
@@ -52,26 +51,24 @@ Within the repositories dashboard, you can view the following metrics, which are
 - **Issues**: This includes the total Opened and Closed Issues for the repositories included in your workspace, as well as the velocity of issues being closed.
 - **Engagement**: This includes the total stars, forks, and activity ratio for the repositories included in your workspace.
 
-
 ## How to Create or Access a Workspace
 
 To create a workspace, start by accessing the **Workspace switcher** in the sidebar. From there, you can create a new workspace or view an existing one.
```

Comment:
To clarify "Workspace switcher", can we say something like this?

```suggestion
To create a workspace, start by accessing the dropdown menu (Workspace switcher) in the sidebar. From there, you can create a new workspace or view an existing one.
```

Reply:
I would rather keep the terminology and add a screenshot with the switcher highlighted. 

This is how Notion has it.

![image](https://github.com/open-sauced/docs/assets/34313413/684696bd-bfce-453b-9337-cf2073e02b9b)